### PR TITLE
kubeadm e2e test for HA

### PR DESCRIPTION
**Pull request by fabriziopandini**
This PR adds a config file for testing kubeadm HA when executing E2E test jobs implemented with kind (kubeadm regular test)

rif https://github.com/kubernetes/kubeadm/issues/1567

NB. In case I'm out of sync with other helping to this effort, feel free to close.

/kind test
/assign @neolit123
/cc @RA489 

/cc @kubernetes/sig-cluster-lifecycle-pr-reviews

### CODE REVIEW

File: tests/e2e/kind/multi-cp/multi-cp.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# a single control-plane cluster with 3 workers
```
Comment (RA489) [inline tests/e2e/kind/multi-cp/multi-cp.yaml]: @fabriziopandini can we substitute single control-plane with multiple control-plane?

Comment (RA489) [reply tests/e2e/kind/multi-cp/multi-cp.yaml]: @fabriziopandini wdyt?

**Comment by RA489** [comment]
@fabriziopandini  PR for it is already in flight 
https://github.com/kubernetes/kubeadm/pull/1571

**Comment by fabriziopandini** [comment]
/close
This seems to be already done

**Comment by RA489** [comment]
@fabriziopandini Thanks!

**Pull request closed without merging**